Code/Editor/BaseLibrary.cpp

```diff
@@ -24,10 +24,10 @@ class CUndoBaseLibrary
     : public IUndoObject
 {
 public:
-    CUndoBaseLibrary(CBaseLibrary* pLib, const QString& description, const QString& selectedItem = 0)
+    CUndoBaseLibrary(CBaseLibrary* pLib, const QString& description, const QString& selectedItem = nullptr)
```

Comment:
suggestion: This looks suspect to me - we don't want to assign a reference to `nullptr`, I think `""` would be a better default if we need to keep a default argument

Reply:
Yup this was mad more visible by replacing the '0' with 'nullptr'